qutebrowser/browser/commands.py

```diff
@@ -1120,7 +1120,8 @@ def quickmark_del(self, name=None):
             raise cmdexc.CommandError("Quickmark '{}' not found!".format(name))
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def bookmark_add(self, url=None, title=None):
+    @cmdutils.argument('toggle', flag='t')
```

Comment:
No need to add this if you don't want to customize the letter - the first one is taken by default.